tests/builders/test_coil_supports.py

```diff
@@ -86,3 +89,90 @@ def test_good_support_radius(self, tf, x_gs):
         assert len(component.get_component("xyz").children) == 1
         assert len(component.get_component("xz").children) == 1
         assert len(component.get_component("xy").children) == 0
+
+
+class TestPFCoilSupportBuilder:
+
+    my_test_params = PFCoilSupportBuilderParams(
+        tf_wp_depth=1.4,
+        tf_wp_width=0.8,
+        tk_tf_side=0.05,
+        pf_s_tk_plate=0.15,
+        pf_s_n_plate=4,
+        pf_s_g=0.05,
+    )
```

Comment:
ParameterFrame needs setting up as in #1942, I will get round to fixing #1947 soon...